Wikipedia/Code/TabsOverviewCoordinator.swift

```diff
@@ -167,19 +171,22 @@ final class TabsOverviewCoordinator: Coordinator {
                 emptyStateTitle: WMFLocalizedString("tabs-empty-view-title", value: "Your tabs will show up here", comment: "Title for the tabs overview screen when there are no tabs"),
                 emptyStateSubtitle: WMFLocalizedString("tabs-empty-view-subtitle", value: "Tap “+” to add tabs to this space or long press an article link to open it in a new tab.", comment: "Subtitle for the tabs overview screen when there are no tabs")
             )
-            
+
+            let dykVM = try? await loadDidYouKnowViewModel()
```

Comment:
We're going to run into the same problem we ran into last time here - this will load did you know and wait for it to finish, which will cause a hang before navigationController?.present is called below.

We should initially instantiate didYouKnowViewModel with nil. Then when the TabsOverview view appears, ask the ArticleTabsViewModel to trigger this closure, which will populate it's own didYouKnowViewModel property. Once that is populated, maybe it could be @Published or something which causes the module to appear on the Tabs Overview view.